var/spack/repos/builtin/packages/py-phylophlan/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyPhylophlan(PythonPackage):
+    """PhyloPhlAn 3.0 is an integrated pipeline for large-scale
+    phylogenetic profiling of genomes and metagenomes."""
+
+    homepage = "https://github.com/biobakery/phylophlan"
+    url      = "https://github.com/biobakery/phylophlan/archive/refs/tags/3.0.2.tar.gz"
```

Comment:
Can we instead download from PyPI? See https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#pypi-vs-github. This will also likely change the checksum. There is a `pypi` variable you can use to simplify the URL.

Reply:
In that case you can leave your build recipe as is. If you want you can open an issue with the phylophlan developers to let them know that their sdists are broken.